pkg/flexvolume/client.go

```diff
@@ -38,18 +60,33 @@ func (c *client) Detach(mountDevice string) error {
 			MountDevice: mountDevice,
 		},
 	)
+
+	output := newOutput(err)
+	if _, osErr := os.Stdout.Write(output); osErr != nil {
```

Comment:
might as well just do:

```go
func writeOutput(err error) {
  return os.Stdout.Write(newOutput(err))
}```

and then instead of doing this logging below (which can be done by the caller), just do:

```go
return writeOutput(err)
```